.github/workflows/companion-deploy.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           context: .
           platforms: linux/amd64,linux/arm64
           file: Dockerfile
-          tags: ${{ steps.docker_meta.outputs.tags }}
+          tags: ${{ env.GITHUB_REF }}
```

Comment:
I think this will become `refs/tags/vX.Y.Z`. Should'nt we instead use the version?